digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/AwsService.scala

```diff
@@ -0,0 +1,299 @@
+package fi.liikennevirasto.digiroad2.service
+
+import fi.liikennevirasto.digiroad2.AssetService
+import fi.liikennevirasto.digiroad2.asset._
+import fi.liikennevirasto.digiroad2.dao.AwsDao
+import fi.liikennevirasto.digiroad2.linearasset.{DynamicAssetValue, DynamicValue, NewLinearAsset, NumericValue, RoadLink, Value}
+import fi.liikennevirasto.digiroad2.service.linearasset._
+import fi.liikennevirasto.digiroad2.service.pointasset.{IncomingObstacle, ObstacleService, PavedRoadService, PedestrianCrossingService, RailwayCrossingService, TrafficLightService, HeightLimit => _, WidthLimit => _}
+import org.json4s._
+import org.json4s.jackson.Serialization
+import org.json4s.jackson.Serialization.write
+import fi.liikennevirasto.digiroad2.client.vvh.VVHClient
+import fi.liikennevirasto.digiroad2.oracle.OracleDatabase
+
+case class Dataset(datasetId: String, geoJson: Map[String, Any], roadlinks: List[List[BigInt]])
+
+class AwsService(vvhClient: VVHClient,
+                 val onOffLinearAssetService: OnOffLinearAssetService,
+                 val roadLinkService: RoadLinkService,
+                 val linearAssetService: LinearAssetService,
+                 val speedLimitService: SpeedLimitService,
+                 val pavedRoadService: PavedRoadService,
+                 val roadWidthService: RoadWidthService,
+                 val manoeuvreService: ManoeuvreService,
+                 val assetService: AssetService,
+                 val obstacleService: ObstacleService,
+                 val pedestrianCrossingService: PedestrianCrossingService,
+                 val railwayCrossingService: RailwayCrossingService,
+                 val trafficLightService: TrafficLightService,
+                 val massTransitLaneService: MassTransitLaneService,
+                 val numberOfLanesService: NumberOfLanesService
+                ) {
+
+  def withDynTransaction[T](f: => T): T = OracleDatabase.withDynTransaction(f)
+  def withDynSession[T](f: => T): T = OracleDatabase.withDynSession(f)
+  def awsDao: AwsDao = new AwsDao
+
+  implicit val formats = Serialization.formats(NoTypeHints)
+
+  final val AwsUser = "AwsUpdater"
+
+  val allDatasetStatus = Map(
+    0 -> "Inserted successfuly",
+    1 -> "Amount of features and roadlinks do not match",
+    2 -> "Errors in the features",
+    3 -> "Processed successfuly",
+    4 -> "Errors while processing"
+  )
+
+  val allFeatureStatus = Map(
+    0 -> "Inserted successfuly",
+    1 -> "Errors while validating",
+    2 -> "Processed successfuly",
+    3 -> "SpeedLimit with invalid speed",
+    4 -> "PavementClass with invalid pavement class",
+    5 -> "Obstacle with invalid class",
+    6 -> "Invalid sideCode"
+  )
```

Comment:
This maps for the error messages would be better if transformed into something like this:

```
sealed trait DatasetStatus{
  def value: Int
}

object DatasetStatus{
  val values = Set[DatasetStatus](Inserted, FeatureRoadlinksDontMatch, ErrorsFeatures, Processed, ErrorsProcessing )

  def apply(intValue: Int): DatasetStatus= {
    values.find(_.value == intValue).getOrElse(ErrorsProcessing)
  }

  case object Inserted extends DatasetStatus{ def value = 0; def description = "Inserted successfuly";}
  case object FeatureRoadlinksDontMatch extends DatasetStatus{ def value = 1; def description = "Amount of features and roadlinks do not match";}
  case object ErrorsFeatures extends DatasetStatus{ def value = 2; def description = "Errors in the features"; }
  case object Processed extends DatasetStatus{ def value = 3; def description = "Processed successfuly"; }
  case object ErrorsProcessing extends DatasetStatus{ def value = 4; def description = "Errors while processing";  }
}
```

and another for the features